internal/uvm/types.go

```diff
@@ -87,11 +88,12 @@ type UtilityVM struct {
 	vpmemDevicesMultiMapped [MaxVPMEMCount]*vPMemInfoMulti
 
 	// SCSI devices that are mapped into a Windows or Linux utility VM
-	scsiLocations       [4][64]*SCSIMount // Hyper-V supports 4 controllers, 64 slots per controller. Limited to 1 controller for now though.
-	scsiControllerCount uint32            // Number of SCSI controllers in the utility VM
-	encryptScratch      bool              // Enable scratch encryption
+	SCSIManager         *scsi.Manager
+	scsiControllerCount uint32 // Number of SCSI controllers in the utility VM
+	reservedSCSISlots   []scsi.Slot
```

Comment:
if these are only needed so we can pass them to `scsi.NewManager(`, can we, in `scis`, define:
```go
type ManagerOptions struct {
  ControllerCount  uint32
  ReservedSlots    []Slot
  // ...
}

// ...

func NewManager(
	attacher Attacher,
	mounter Mounter,
	unplugger Unplugger,
	opts *ManagerOpts,
) *Manager {
  // ...
```

that way its cleaner/more apparent here in `uvm` that these fields are just options used to create the `scsi.Manager`

Reply:
All of the parameters to `NewManager`, except for `reservedSlots`, are required (e.g. need a non-zero-value). I'm not sure how much it would add to move these to a separate struct, since in practice the client would need to declare almost all of them anyways. Moving just `reservedSlots` to a separate struct makes some sense to me, but I'm not sure it's worth it for only one field. If we had more things that were truly optional, it would make more sense to do this.